test/ui-test/src/screens/StatusCommunityScreen.py

```diff
@@ -465,4 +476,37 @@ def click_sidebar_option(self, community_sidebar_option:str):
     def verify_option_exists(self, option:str):
         if option=="Permissions":
             title = get_obj(CommunitySettingsComponents.PERMISSIONS_BUTTON.value).title
-            verify_text(option, str(title))    
+            verify_text(option, str(title))
+              
+    def select_community_settings_option(self, option:str):
+        if option=="Permissions":
+            click_obj_by_name(CommunitySettingsComponents.PERMISSIONS_BUTTON.value)
+     
+    def verify_permission_screen_title(self, option:str):
+        if option=="Permissions":
+            title = get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_TITLE.value).text
+            verify_text(option, str(title))
+             
+    def verify_welcome_permission_image(self):
+        path = get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_IMAGE.value).source.path
+        verify_text_contains(str(path), "permissions2_3")
+    
+    def verify_welcome_settings_title(self):
+        verify_equals(get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_SETTINGS_TITLE.value).text, "Permissions")    
+    
+    def verify_welcome_settings_subtitle(self):
+        verify_equals(get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_SETTINGS_SUBTITLE.value).text, "You can manage your community by creating and issuing membership and access permissions")
+        
+    def verify_welcome_settings_checklist(self):
+        checklist = []
+        checklist.append(get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_SETTINGS_CHECKLIST_ELEMENT1.value).text)
+        checklist.append(get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_SETTINGS_CHECKLIST_ELEMENT2.value).text)
+        checklist.append(get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_SETTINGS_CHECKLIST_ELEMENT3.value).text)
+
+        verify_equals(checklist[0], "Give individual members access to private channels")
+        verify_equals(checklist[1], "Monetise your community with subscriptions and fees")
+        verify_equals(checklist[2], "Require holding a token or NFT to obtain exclusive membership rights")
```

Comment:
In my opinion, testing text can be flimsy. If the text is changed, we have to update the tests as well. Maybe just checking if the text obj is there and visible is enough?

Reply:
I think that in some point we will also need to check the texts and also text in different languages so.. But maybe an improvement here would be to add it as a param in the BDD feature file!

Reply:
Sounds good